packages/webpack-cli/package.json

```diff
@@ -18,7 +18,6 @@
   "keywords": [
     "webpack",
     "cli",
-    "scaffolding",
```

Comment:
we actually do scaffold things using init/loader/plugins, just not with webpack-scaffold

Reply:
I agree here

Reply:
Hmm, If we think closely it's actually another package `@webpack-cli/generators` for the scaffolding stuff. But I think we are good here. I don't have a strong opinion on this one.

Reply:
scaffolding is meant to be done through CLI only, so we should keep this keyword and should not be removed.